.golangci.yml

```diff
@@ -119,6 +119,8 @@ linters:
     - revive
     - paralleltest
     - gocognit
+    - tagalign
+    - ireturn
```

Comment:
@ademidoff 
`ireturn` had fixed with `nolintlint` https://github.com/percona/pmm/pull/2545/commits/233abb65fac8e33d6ae72569507bca381f459c18

The `tagalign` issue can fix automatically with this tool.
https://github.com/4meepo/tagalign

But it can be conflicted the rule of `lll` after formatting `tagalign`.

```
% golangci-lint run --out-format line-number -c=.golangci.yml
admin/cli/flags/global.go:30:38: tag is not aligned, should be: help:"PMM Server URL in https://username:password@pmm-server-host/ format" placeholder:"SERVER-URL" (tagalign)
	ServerURL               *url.URL    `placeholder:"SERVER-URL" help:"PMM Server URL in https://username:password@pmm-server-host/ format"`
	                                    ^
admin/cli/flags/global.go:31:38: tag is not aligned, should be: help:"Skip PMM Server TLS certificate validation"                          name:"server-insecure-tls" (tagalign)
	SkipTLSCertificateCheck bool        `name:"server-insecure-tls" help:"Skip PMM Server TLS certificate validation"`
	                                    ^
admin/cli/flags/global.go:32:38: tag is not aligned, should be: help:"Enable debug logging"                                                name:"debug" (tagalign)
	EnableDebug             bool        `name:"debug" help:"Enable debug logging"`
	                                    ^
admin/cli/flags/global.go:33:38: tag is not aligned, should be: help:"Enable trace logging (implies debug)"                                name:"trace" (tagalign)
	EnableTrace             bool        `name:"trace" help:"Enable trace logging (implies debug)"`
	                                    ^
admin/cli/flags/global.go:34:38: tag is not aligned, should be: default:"${defaultListenPort}"                                             help:"Set listen port of pmm-agent" (tagalign)
	PMMAgentListenPort      uint32      `default:"${defaultListenPort}" help:"Set listen port of pmm-agent"`
	                                    ^
admin/cli/flags/global.go:36:38: tag is not aligned, should be: help:"Show application version"                                            short:"v" (tagalign)
	Version                 versionFlag `short:"v" help:"Show application version"`
	                                    ^
vmproxy/main.go:38:25: tag is not aligned, should be: default:"1280"                  help:"Listen port for proxy" (tagalign)
	ListenPort    int      `default:"1280" help:"Listen port for proxy"`
	                       ^
vmproxy/main.go:39:25: tag is not aligned, should be: default:"127.0.0.1"             help:"Listen address for proxy" (tagalign)
	ListenAddress string   `default:"127.0.0.1" help:"Listen address for proxy"`
	                       ^
vmproxy/main.go:40:25: tag is not aligned, should be: default:"X-Proxy-Filter"        help:"Header name to read filter configuration from. The content of the header shall be a base64 encoded JSON array with strings. Each string is a filter. Multiple filters are joined with a logical OR." (tagalign)
	HeaderName    string   `default:"X-Proxy-Filter" help:"Header name to read filter configuration from. The content of the header shall be a base64 encoded JSON array with strings. Each string is a filter. Multiple filters are joined with a logical OR."` //nolint:lll
	                       ^
admin/commands/annotation.go:48:23: tag is not aligned, should be: arg:""                                                                    help:"Text of annotation" (tagalign)
	Text        string   `arg:"" help:"Text of annotation"`
	                     ^
admin/commands/config.go:46:29: tag is not aligned, should be: arg:""                                                                                default:"${nodeIp}"                                                                                                help:"Node address (autodetected default: ${nodeIp})" (tagalign)
	NodeAddress       string   `arg:"" default:"${nodeIp}" help:"Node address (autodetected default: ${nodeIp})"`
	                           ^
admin/commands/config.go:47:29: tag is not aligned, should be: arg:""                                                                                default:"${nodeTypeDefault}"                                                                                       enum:"generic,container"                                                                                                                                     help:"Node type, one of: generic, container (default: ${nodeTypeDefault})" (tagalign)
	NodeType          string   `arg:"" enum:"generic,container" default:"${nodeTypeDefault}" help:"Node type, one of: generic, container (default: ${nodeTypeDefault})"`
	                           ^
admin/commands/config.go:48:29: tag is not aligned, should be: arg:""                                                                                default:"${hostname}"                                                                                              help:"Node name (autodetected default: ${hostname})" (tagalign)
	NodeName          string   `arg:"" default:"${hostname}" help:"Node name (autodetected default: ${hostname})"`
	                           ^
admin/commands/config.go:54:29: tag is not aligned, should be: default:"auto"                                                                        enum:"${metricsModesEnum}"                                                                                         help:"Metrics flow mode for agents node-exporter, can be push - agent will push metrics, pull - server scrape metrics from agent or auto - chosen by server" (tagalign)
	MetricsMode       string   `enum:"${metricsModesEnum}" default:"auto" help:"Metrics flow mode for agents node-exporter, can be push - agent will push metrics, pull - server scrape metrics from agent or auto - chosen by server"`
	                           ^
admin/commands/config.go:56:29: tag is not aligned, should be: help:"Custom user-assigned labels"                                                    placeholder:"KEY=VALUE,KEY=VALUE,..." (tagalign)
	CustomLabels      string   `placeholder:"KEY=VALUE,KEY=VALUE,..." help:"Custom user-assigned labels"`
	                           ^
admin/commands/config.go:57:29: tag is not aligned, should be: help:"Base path where all binaries, tools and collectors of PMM client are located"   name:"paths-base" (tagalign)
	BasePath          string   `name:"paths-base" help:"Base path where all binaries, tools and collectors of PMM client are located"`
	                           ^
admin/commands/config.go:58:29: tag is not aligned, should be: default:"warn"                                                                        enum:"debug,info,warn,error,fatal"                                                                                 help:"Logging level" (tagalign)
	LogLevel          string   `enum:"debug,info,warn,error,fatal" default:"warn" help:"Logging level"`
	                           ^
admin/commands/config.go:59:29: tag is not aligned, should be: default:"1024"                                                                        help:"Take and return N most recent log lines in logs.zip for each: server, every configured exporters and agents" (tagalign)
	LogLinesCount     uint     `help:"Take and return N most recent log lines in logs.zip for each: server, every configured exporters and agents" default:"1024"`
	                           ^
admin/commands/status.go:92:24: tag is not aligned , should be: help:"Time to wait for a successful response from pmm-agent" name:"wait" (tagalign)
	Timeout time.Duration `name:"wait" help:"Time to wait for a successful response from pmm-agent"`
	                      ^
admin/commands/summary.go:357:20: tag is not aligned, should be: help:"Include performance profiling data"     name:"pprof" (tagalign)
	Pprof      bool   `name:"pprof" help:"Include performance profiling data"`
	                  ^
admin/commands/pmm/client/install.go:27:22: tag is not aligned, should be: default:"/usr/local/percona/pmm2"                       help:"Path where PMM Client shall be installed" (tagalign)
	InstallPath  string `default:"/usr/local/percona/pmm2" help:"Path where PMM Client shall be installed"`
	                    ^
admin/commands/pmm/client/install.go:30:22: tag is not aligned, should be: help:"PMM Server version to install (default: latest)"  name:"use-version" (tagalign)
	Version      string `name:"use-version" help:"PMM Server version to install (default: latest)"`
	                    ^
admin/commands/pmm/client/upgrade.go:35:22: tag is not aligned, should be: default:"autodetect"                                      enum:"autodetect,package-manager,tarball,docker" help:"Type of PMM Client distribution. One of: autodetect,package-manager,tarball,docker" (tagalign)
	Distribution string `enum:"autodetect,package-manager,tarball,docker" default:"autodetect" help:"Type of PMM Client distribution. One of: autodetect,package-manager,tarball,docker"` //nolint:lll
	                    ^
admin/commands/pmm/client/upgrade.go:36:22: tag is not aligned, should be: help:"PMM Client version to upgrade to (default: latest)" name:"use-version" (tagalign)
	Version      string `name:"use-version" help:"PMM Client version to upgrade to (default: latest)"`
	                    ^
admin/commands/pmm/client/upgrade.go:38:22: tag is not aligned, should be: default:"/usr/local/percona/pmm2" group:"Tarball flags"                                   help:"Path where PMM Client is installed" (tagalign)
	InstallPath  string `group:"Tarball flags" default:"/usr/local/percona/pmm2" help:"Path where PMM Client is installed"`
	                    ^
admin/commands/pmm/client/upgrade.go:39:22: tag is not aligned, should be: group:"Tarball flags"             help:"Set file ownership instead of the current user" (tagalign)
	User         string `group:"Tarball flags" help:"Set file ownership instead of the current user"`
	                    ^
admin/commands/pmm/client/upgrade.go:40:22: tag is not aligned, should be: group:"Tarball flags"             help:"Set group ownership instead of the current group" (tagalign)
	Group        string `group:"Tarball flags" help:"Set group ownership instead of the current group"`
	                    ^
admin/commands/pmm/client/upgrade.go:41:22: tag is not aligned, should be: group:"Tarball flags"             help:"Skip checksum validation of the downloaded files" (tagalign)
	SkipChecksum bool   `group:"Tarball flags" help:"Skip checksum validation of the downloaded files"`
	                    ^
admin/commands/management/add.go:26:50: tag is not aligned, should be: cmd:"" help:"Add HAProxy to monitoring"                                                                name:"haproxy" (tagalign)
	HAProxy            AddHAProxyCommand            `cmd:"" name:"haproxy" help:"Add HAProxy to monitoring"`
	                                                ^
admin/commands/management/add.go:27:50: tag is not aligned, should be: cmd:"" help:"Add MongoDB to monitoring"                                                                name:"mongodb" (tagalign)
	MongoDB            AddMongoDBCommand            `cmd:"" name:"mongodb" help:"Add MongoDB to monitoring"`
	                                                ^
admin/commands/management/add.go:28:50: tag is not aligned, should be: cmd:"" help:"Add MySQL to monitoring"                                                                  name:"mysql" (tagalign)
	MySQL              AddMySQLCommand              `cmd:"" name:"mysql" help:"Add MySQL to monitoring"`
	                                                ^
admin/commands/management/add.go:29:50: tag is not aligned, should be: cmd:"" help:"Add PostgreSQL to monitoring"                                                             name:"postgresql" (tagalign)
	PostgreSQL         AddPostgreSQLCommand         `cmd:"" name:"postgresql" help:"Add PostgreSQL to monitoring"`
	                                                ^
admin/commands/management/add.go:30:50: tag is not aligned, should be: cmd:"" help:"Add ProxySQL to monitoring"                                                               name:"proxysql" (tagalign)
	ProxySQL           AddProxySQLCommand           `cmd:"" name:"proxysql" help:"Add ProxySQL to monitoring"`
	                                                ^
admin/commands/management/add.go:35:28: tag is not aligned, should be: help:"Service name (overrides positional argument)"                   name:"service-name" placeholder:"NAME" (tagalign)
	AddServiceNameFlag string `name:"service-name" placeholder:"NAME" help:"Service name (overrides positional argument)"`
	                          ^
admin/commands/management/add.go:36:28: tag is not aligned, should be: help:"Service hostname or IP address (overrides positional argument)" name:"host"         placeholder:"HOST" (tagalign)
	AddHostFlag        string `name:"host" placeholder:"HOST" help:"Service hostname or IP address (overrides positional argument)"`
	                          ^
admin/commands/management/add.go:37:28: tag is not aligned, should be: help:"Service port number (overrides positional argument)"            name:"port"         placeholder:"PORT" (tagalign)
	AddPortFlag        uint16 `name:"port" placeholder:"PORT" help:"Service port number (overrides positional argument)"`
	                          ^
admin/commands/management/add.go:42:21: tag is not aligned , should be: default:"warn" enum:"debug,info,warn,error,fatal" help:"Service logging level. One of: [debug, info, warn, error, fatal]" name:"log-level" (tagalign)
	AddLogLevel string `name:"log-level" enum:"debug,info,warn,error,fatal" default:"warn" help:"Service logging level. One of: [debug, info, warn, error, fatal]"`
	                   ^
admin/commands/management/add.go:47:21: tag is not aligned , should be: default:"warn" enum:"debug,info,warn,error" help:"Service logging level. One of: [debug, info, warn, error]" name:"log-level" (tagalign)
	AddLogLevel string `name:"log-level" enum:"debug,info,warn,error" default:"warn" help:"Service logging level. One of: [debug, info, warn, error]"`
	                   ^
admin/commands/management/add_external.go:58:40: tag is not aligned, should be: default:"${hostname}${externalDefaultServiceName}"                       help:"Service name (autodetected default: ${hostname}${externalDefaultServiceName})" (tagalign)
	ServiceName         string            `default:"${hostname}${externalDefaultServiceName}" help:"Service name (autodetected default: ${hostname}${externalDefaultServiceName})"`
	                                      ^
admin/commands/management/add_external.go:59:40: tag is not aligned, should be: help:"Node ID where agent runs (default is autodetected)"                name:"agent-node-id" (tagalign)
	RunsOnNodeID        string            `name:"agent-node-id" help:"Node ID where agent runs (default is autodetected)"`
	                                      ^
admin/commands/management/add_external.go:62:40: tag is not aligned, should be: help:"Credentials provider"                                              type:"existingfile" (tagalign)
	CredentialsSource   string            `type:"existingfile" help:"Credentials provider"`
	                                      ^
admin/commands/management/add_external.go:63:40: tag is not aligned, should be: help:"Scheme to generate URI to exporter metrics endpoints"              placeholder:"http or https" (tagalign)
	Scheme              string            `placeholder:"http or https" help:"Scheme to generate URI to exporter metrics endpoints"`
	                                      ^
admin/commands/management/add_external.go:64:40: tag is not aligned, should be: help:"Path under which metrics are exposed, used to generate URI"        placeholder:"/metrics" (tagalign)
	MetricsPath         string            `placeholder:"/metrics" help:"Path under which metrics are exposed, used to generate URI"`
	                                      ^
admin/commands/management/add_external.go:65:40: tag is not aligned, should be: help:"Listen port of external exporter for scraping metrics. (Required)" placeholder:"port"                                                                   required:"" (tagalign)
	ListenPort          uint16            `placeholder:"port" required:"" help:"Listen port of external exporter for scraping metrics. (Required)"`
	                                      ^
admin/commands/management/add_external.go:66:40: tag is not aligned, should be: help:"Node ID where service runs (default is autodetected)"              name:"service-node-id" (tagalign)
	NodeID              string            `name:"service-node-id" help:"Node ID where service runs (default is autodetected)"`
	                                      ^
admin/commands/management/add_external.go:67:40: tag is not aligned, should be: help:"Environment name like 'production' or 'qa'"                        placeholder:"prod" (tagalign)
	Environment         string            `placeholder:"prod" help:"Environment name like 'production' or 'qa'"`
	                                      ^
admin/commands/management/add_external.go:68:40: tag is not aligned, should be: help:"Cluster name"                                                      placeholder:"east-cluster" (tagalign)
	Cluster             string            `placeholder:"east-cluster" help:"Cluster name"`
	                                      ^
admin/commands/management/add_external.go:69:40: tag is not aligned, should be: help:"Replication set name"                                              placeholder:"rs1" (tagalign)
	ReplicationSet      string            `placeholder:"rs1" help:"Replication set name"`
	                                      ^
admin/commands/management/add_external.go:70:40: tag is not aligned, should be: help:"Custom user-assigned labels"                                       mapsep:"" (tagalign)
	CustomLabels        map[string]string `mapsep:"," help:"Custom user-assigned labels"`
	                                      ^
admin/commands/management/add_external.go:71:40: tag is not aligned, should be: default:"auto"                                                           enum:"${metricsModesEnum}"                                                           help:"Metrics flow mode, can be push - agent will push metrics, pull - server scrape metrics from agent or auto - chosen by server" (tagalign)
	MetricsMode         string            `enum:"${metricsModesEnum}" default:"auto" help:"Metrics flow mode, can be push - agent will push metrics, pull - server scrape metrics from agent or auto - chosen by server"`
	                                      ^
% tagalign -fix -sort ./...
...
% golangci-lint run --out-format line-number -c=.golangci.yml
admin/commands/pmm/server/docker/upgrade.go:39: line is 272 characters (lll)
	DockerImage            string `default:"percona/pmm-server:2"                                                                                                                                  help:"Docker image to use to upgrade PMM Server. Defaults to latest version"`
admin/commands/pmm/server/docker/upgrade.go:40: line is 244 characters (lll)
	ContainerID            string `default:"pmm-server"                                                                                                                                            help:"Container ID of the PMM Server to upgrade"`
admin/commands/pmm/server/docker/upgrade.go:42: line is 258 characters (lll)
	NewContainerNamePrefix string `default:"pmm-server"                                                                                                                                            help:"Prefix for the name of the new container for PMM Server"`
admin/commands/pmm/server/docker/upgrade.go:43: line is 282 characters (lll)
	AssumeYes              bool   `help:"Assume yes for all prompts"                                                                                                                               name:"yes"                                                                   short:"y"`
admin/commands/inventory/add_agent_node_exporter.go:46: line is 180 characters (lll)
	PMMAgentID        string            `arg:""                                                                            help:"The pmm-agent identifier which runs this instance"`
admin/commands/inventory/add_agent_node_exporter.go:50: line is 245 characters (lll)
	LogLevel          string            `default:"warn"                                                                    enum:"debug,info,warn,error"                             help:"Service logging level. One of: [debug, info, warn, error]"`
admin/commands/inventory/add_agent_postgres_exporter.go:50: line is 182 characters (lll)
	PMMAgentID          string            `arg:""                                                                            help:"The pmm-agent identifier which runs this instance"`
admin/commands/inventory/add_agent_postgres_exporter.go:52: line is 194 characters (lll)
	Username            string            `arg:""                                                                            help:"PostgreSQL username for scraping metrics"          optional:""`
admin/commands/inventory/add_agent_postgres_exporter.go:64: line is 247 characters (lll)
	LogLevel            string            `default:"warn"                                                                    enum:"debug,info,warn,error"                             help:"Service logging level. One of: [debug, info, warn, error]"`
admin/commands/inventory/add_agent_proxysql_exporter.go:50: line is 182 characters (lll)
	PMMAgentID          string            `arg:""                                                                            help:"The pmm-agent identifier which runs this instance"`
admin/commands/inventory/add_agent_proxysql_exporter.go:52: line is 194 characters (lll)
	Username            string            `arg:""                                                                            help:"ProxySQL username for scraping metrics"            optional:""`
admin/commands/inventory/add_agent_proxysql_exporter.go:61: line is 254 characters (lll)
	LogLevel            string            `default:"warn"                                                                    enum:"debug,info,warn,error,fatal"                       help:"Service logging level. One of: [debug, info, warn, error, fatal]"`
admin/commands/inventory/add_agent_qan_mysql_perfschema_agent.go:57: line is 174 characters (lll)
	PMMAgentID           string            `arg:""                                                                   help:"The pmm-agent identifier which runs this instance"`
admin/commands/inventory/add_agent_qan_mysql_perfschema_agent.go:59: line is 186 characters (lll)
	Username             string            `arg:""                                                                   help:"MySQL username for scraping metrics"               optional:""`
admin/commands/inventory/add_agent_qan_mysql_perfschema_agent.go:63: line is 245 characters (lll)
	CommentsParsing      string            `default:"off"                                                            enum:"on,off"                                            help:"Enable/disable parsing comments from queries. One of: [on, off]"`
admin/commands/inventory/add_agent_qan_mysql_perfschema_agent.go:71: line is 246 characters (lll)
	LogLevel             string            `default:"warn"                                                           enum:"debug,info,warn,error,fatal"                       help:"Service logging level. One of: [debug, info, warn, error, fatal]"`
admin/commands/inventory/add_agent_qan_postgres_pgstatements_agent.go:49: line is 173 characters (lll)
	PMMAgentID          string            `arg:""                                                                   help:"The pmm-agent identifier which runs this instance"`
admin/commands/inventory/add_agent_qan_postgres_pgstatements_agent.go:51: line is 185 characters (lll)
	Username            string            `arg:""                                                                   help:"PostgreSQL username for QAN agent"                 optional:""`
admin/commands/inventory/add_agent_qan_postgres_pgstatements_agent.go:55: line is 244 characters (lll)
	CommentsParsing     string            `default:"off"                                                            enum:"on,off"                                            help:"Enable/disable parsing comments from queries. One of: [on, off]"`
admin/commands/inventory/add_agent_qan_postgres_pgstatements_agent.go:62: line is 245 characters (lll)
	LogLevel            string            `default:"warn"                                                           enum:"debug,info,warn,error,fatal"                       help:"Service logging level. One of: [debug, info, warn, error, fatal]"`
admin/commands/inventory/add_agent_qan_postgres_pgstatmonitor_agent.go:50: line is 175 characters (lll)
	PMMAgentID            string            `arg:""                                                                   help:"The pmm-agent identifier which runs this instance"`
admin/commands/inventory/add_agent_qan_postgres_pgstatmonitor_agent.go:52: line is 187 characters (lll)
	Username              string            `arg:""                                                                   help:"PostgreSQL username for QAN agent"                 optional:""`
admin/commands/inventory/add_agent_qan_postgres_pgstatmonitor_agent.go:56: line is 246 characters (lll)
	CommentsParsing       string            `default:"off"                                                            enum:"on,off"                                            help:"Enable/disable parsing comments from queries. One of: [on, off]"`
admin/commands/inventory/add_agent_qan_postgres_pgstatmonitor_agent.go:64: line is 247 characters (lll)
	LogLevel              string            `default:"warn"                                                           enum:"debug,info,warn,error,fatal"                       help:"Service logging level. One of: [debug, info, warn, error, fatal]"`
admin/commands/inventory/add_agent_rds_exporter.go:49: line is 185 characters (lll)
	PMMAgentID             string            `arg:""                                                                            help:"The pmm-agent identifier which runs this instance"`
admin/commands/inventory/add_agent_rds_exporter.go:58: line is 257 characters (lll)
	LogLevel               string            `default:"warn"                                                                    enum:"debug,info,warn,error,fatal"                       help:"Service logging level. One of: [debug, info, warn, error, fatal]"`
admin/commands/management/add.go:26: line is 171 characters (lll)
	HAProxy            AddHAProxyCommand            `cmd:"" help:"Add HAProxy to monitoring"                                                                name:"haproxy"`
admin/commands/management/add.go:27: line is 171 characters (lll)
	MongoDB            AddMongoDBCommand            `cmd:"" help:"Add MongoDB to monitoring"                                                                name:"mongodb"`
admin/commands/management/add.go:29: line is 174 characters (lll)
	PostgreSQL         AddPostgreSQLCommand         `cmd:"" help:"Add PostgreSQL to monitoring"                                                             name:"postgresql"`
admin/commands/management/add.go:30: line is 172 characters (lll)
	ProxySQL           AddProxySQLCommand           `cmd:"" help:"Add ProxySQL to monitoring"                                                               name:"proxysql"`
admin/commands/management/add_external_serverless.go:65: line is 190 characters (lll)
	Group               string            `default:"${externalDefaultGroupExporter}"                         help:"Group name of external service (default: ${externalDefaultGroupExporter})"`
```